test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -81,12 +81,15 @@ def run_ort(ort_sess, input):
     return inline_flatten_list(ort_outs, [])
 
 
-def ort_compare_with_pytorch(ort_outs, output, rtol, atol):
+def ort_compare_with_pytorch(ort_outs, output, rtol, atol, check_dtypes):
```

Comment:
To pass the failing checks, I think you can add a default value for `check_dtypes`. Default `True` would be ideal but `False` is fine if that causes test failures elsewhere.